src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/MvcViewDocumentClassifierPass.cs

```diff
@@ -37,5 +48,22 @@ public class MvcViewDocumentClassifierPass : DocumentClassifierPassBase
             method.Modifiers.Add("override");
             method.ReturnType = $"global::{typeof(System.Threading.Tasks.Task).FullName}";
         }
+
+        private static string BytesToString(byte[] bytes)
+        {
+            if (bytes == null)
+            {
+                throw new ArgumentNullException(nameof(bytes));
+            }
+
+            var result = new StringBuilder(bytes.Length);
+            for (var i = 0; i < bytes.Length; i++)
+            {
+                // The x2 format means lowercase hex, where each byte is a 2-character string.
+                result.Append(bytes[i].ToString("x2"));
+            }
+
+            return result.ToString();
```

Comment:
Don't we already have this code elsewhere?

Reply:
Yup. But we don't have the IVT. So I thought if we have to duplicate in one place, I'll just duplicate it everywhere

Reply:
Shared file? We now have 4 copies of this code :(